packages/cli/src/applyPreset.ts

```diff
@@ -90,6 +90,3 @@ function valueOfArg(argName: string): string | null {
 
   return null;
 }
-
-// Add empty export to make this a module
-export {};
```

Comment:
according to the comment these exports seem to have a purpose

Reply:
Id there is an import, we don't need an export to make it a module.